modules/client/src/default.ts

```diff
@@ -0,0 +1,46 @@
+import { ConnextStore } from "@connext/store";
+import { ethers } from "ethers";
+
+import { ClientOptions } from "./types";
+
+export function generateMnemonic(): string {
+  const entropy = ethers.utils.randomBytes(16);
+  const mnemonic = ethers.utils.HDNode.entropyToMnemonic(entropy);
+  return mnemonic;
+}
+
+export async function getDefaultMnemonic(store: ConnextStore): Promise<string> {
+  const MNEMONIC_KEY = "MNEMONIC";
+  let mnemonic = await store.get(MNEMONIC_KEY);
+  if (!mnemonic) {
+    mnemonic = generateMnemonic();
+    store.set([{ path: MNEMONIC_KEY, value: mnemonic }]);
+  }
+  return mnemonic;
+}
```

Comment:
seems strange to have default options that aren't secure

Reply:
Perhaps default is not right word here, this is more like a beginner options for hackathons and demos, realistically we would advise production apps to manage their own mnemonics

However you can still pass override options while maintain the network string, here is an example for a react-native wallet
```javascript
const channel = await connext.connect('mainnet', {
  xpub,
  keygen,
 asyncStorage
})
```
Obviously the network string is not intended for production use but it can still be used to abstract the nodeUrl and ethProviderUrl to make it easier for us to bootstrap integrations without many breaking changes for our users

Reply:
yeah, we should make sure we add tests for all of this stuff.

i also think we should disable the use of default mnemonics for mainnet

Reply:
Pushed these changes 👍